### ISSUE TITLE
[Feature] BigQuery source needed [Belvedere] 

### ISSUE THREAD

**Issue by mboyanna-sn** [streamnative/pulsar-io-bigquery#62]
@zenbound commented on [Wed Jun 22 2022](https://github.com/streamnative/cloud-support-tickets/issues/86)

## Customer Info

- Organization Name: Belvedere

## Describe the issue

Referencing https://streamnative.io/blog/engineering/2022-02-03-integrating-apache-pulsar-with-bigquery/

"Big Query source for local development and possibly backtesting in the future."

Rather than sink, the customer Belvedere would like to utilize source functionality. Currently, we only seem to support sink, unless there is some undocumented functionality. 

## Additional context

https://streamnative.slack.com/archives/CVDQE4338/p1654735370062589?thread_ts=1654735230.727459&cid=CVDQE4338

https://support.streamnative.io/a/tickets/2370

## Urgency / Deadline

End of Q3 deployment

---

@sijie commented on [Wed Jun 22 2022](https://github.com/streamnative/cloud-support-tickets/issues/86#issuecomment-1163405967)

<div>Agent Jeris Seth Sundell linked Freshdesk ticket <a href="https://streamnative.freshdesk.com/a/tickets/2370">2370</a> for this issue.</div>

---

@sijie commented on [Wed Jun 22 2022](https://github.com/streamnative/cloud-support-tickets/issues/86#issuecomment-1163407181)

<div>Agent Jeris Seth Sundell created comment. </div></br><div style="font-family:-apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Arial, sans-serif; font-size:14px">
<div dir="ltr">Hi Jesse Kleve,<br><br>Your reference number is feature request #86 on this issue. I will help prioritize and keep this ticket on hold while we discuss and plan internally. <br><br>
</div>
<div dir="ltr"><br></div>
<div>Best regards,</div>
<div>
<span style="font-size: 18px">Jeris Seth Sundell</span><br><span style="color: rgb(55, 155, 255)">Customer Reliability Engineer</span><br><span style="color: rgb(55, 155, 255)"><a href="https://streamnative.io" style="color: inherit" rel="noreferrer">streamnative.io</a> </span><span style="color: rgb(0, 0, 0)">|</span><span style="color: rgb(55, 155, 255)"> <a href="http://github.com/streamnative" style="color: inherit" target="_blank" rel="noreferrer"><img src="https://github.com/lmnts-dev/image-hosting/blob/master/streamnative-social.png?raw=true" width="20" height="20" alt="" style="max-width: 100%" class="fr-fic fr-dii"></a><a href="https://www.linkedin.com/company/streamnative/" style="color: inherit" target="_blank" rel="noreferrer"><img src="https://github.com/lmnts-dev/image-hosting/blob/master/streamnative-linkedin.png?raw=true" width="21" height="20" style="max-width: 100%" class="fr-fic fr-dii"></a><img src="https://github.com/lmnts-dev/image-hosting/blob/master/streamnative-twitter.png?raw=true" width="21" height="20" style="max-width: 100%" class="fr-fic fr-dii"></span>
</div>
</div>

---

@sijie commented on [Tue Jun 28 2022](https://github.com/streamnative/cloud-support-tickets/issues/86#issuecomment-1169146905)

<div>Agent Jeris Seth Sundell created comment. </div></br><div style="font-family:-apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Arial, sans-serif; font-size:14px">
<div dir="ltr">Hi Jesse Kleve,<br><br>Just a quick check in. The status of this request is still in progress. I will do my best to update you once a week and ping the team that owns this in parallel. <br><br>
</div>
<div dir="ltr">
<img src="https://i.imgur.com/7XXgJPw.png" data-height="105" height="105px" class="fr-fic fr-dii"><br>
</div>
<div>Best regards,</div>
<div>
<span style="font-size: 18px">Jeris Seth Sundell</span><br><span style="color: rgb(55, 155, 255)">Customer Reliability Engineer</span><br><span style="color: rgb(55, 155, 255)"><a href="https://streamnative.io" style="color: inherit" rel="noreferrer">streamnative.io</a> </span><span style="color: rgb(0, 0, 0)">|</span><span style="color: rgb(55, 155, 255)"> <a href="http://github.com/streamnative" style="color: inherit" target="_blank" rel="noreferrer"><img src="https://github.com/lmnts-dev/image-hosting/blob/master/streamnative-social.png?raw=true" width="20" height="20" alt="" style="max-width: 100%" class="fr-fic fr-dii"></a><a href="https://www.linkedin.com/company/streamnative/" style="color: inherit" target="_blank" rel="noreferrer"><img src="https://github.com/lmnts-dev/image-hosting/blob/master/streamnative-linkedin.png?raw=true" width="21" height="20" style="max-width: 100%" class="fr-fic fr-dii"></a><img src="https://github.com/lmnts-dev/image-hosting/blob/master/streamnative-twitter.png?raw=true" width="21" height="20" style="max-width: 100%" class="fr-fic fr-dii"></span>
</div>
</div>

---

@martijngonlag commented on [Mon Jul 11 2022](https://github.com/streamnative/cloud-support-tickets/issues/86#issuecomment-1180920544)

@mboyanna-sn @nlu90, could you please advise whether or not this feature request is something we could support by the end of Q3? If not, that's OK, just need to be able to relay that to the customer sooner rather than later so they can plan accordingly.

**Comment by lin-zhao** [comment]
@sara-hannigan I can' assign this ticket or make updates. What do we need to do?
